web/docs/setup/tune.md

```diff
@@ -54,129 +52,69 @@ The `vast.import.read-timeout` option determines how long a call to read data
-   read-only)
+## Storage Engine
 
-VAST defaults to the `feather` store. Enable the Parquet store by loading the
-plugin and adjusting `vast.store-backend`:
+The central component of VAST's storage engine is the *catalog*. It owns the
+partitions, keeps meta data about them, and maintains a set of sparse secondary
+indexes to identify relevant partitions for a given query. In addition, a
+partition can have a set of dense secondary indexes in addition to the raw data
+stored.
 
-```yaml
-vast:
-  plugins:
-    - parquet
-  store-backend: parquet
-```
-
-There's an inherent space-time tradeoff between the stores that affects CPU,
-memory, and storage characteristics. Compared to the Feather and Segment stores,
-Parquet differs as follows:
-
-1. Parquet files occupy ~40% less space, which also reduces I/O pressure during
-   querying.
-2. Parquet utilizes more CPU cycles during ingest (~10%) and querying.
-
-Parquet has the major advantage that it's the de-facto standard for encoding
-columnar data in modern data architectures. This allows other applications that
-support reading from Parquet *native* access to the data.
+![Multi-Tier Indexing](multi-tier-indexing.excalidraw.svg)
 
-:::tip Recommendation
+The catalog's secondary indexes are space-efficient sketch data structures
+(e.g., Bloom filters, min-max summaries) that have a low memory footprint but
+may yield false positives. VAST keeps all sketches in memory.
 
-Use Parquet when:
+The amount of memory that the storage engine can consume is not explicitly
+configurable, but there exist various options that have a direct impact.
 
-1. Storage is scarce, and you want to increase data retention
-2. Workloads are I/O-bound and you have available CPU
-3. Reading data with with off-the-shelf data science tools is a use case
-:::
+### Control the partition size
 
-VAST supports [rebuilding the entire database](#rebuild-partitions) in case you
-want to switch to a different store format. However, VAST works perfectly fine
-with a mixed-storage configuration, so a full rebuild is not required.
+VAST groups table slices with the same schema in a partition. There exist
+mutable *active partitions* that VAST writes to during ingestion, and immutable
+*passive partitions* that VAST reads from during query execution.
 
-### Compression
+When constructing a partition, the parameter `vast.max-partition-size` sets an
+upper bound on the number of records in a partition, across all table slices.
+The parameter `vast.active-partition-timeout` provides a time-based upper bound:
+once reached, VAST considers the partition as complete, regardless of the number
+of records.
 
-VAST compresses partitions using Zstd for partitions at rest. To fine-tune
-the space-time trade-off, VAST offers a setting, `vast.zstd-compression-level`
-to allow fine-tuning the compression level:
+The two parameters are decoupled to allow for independent control of throughput
+and freshness.
```

Comment:
This should also mention that VAST automatically merges undersized partitions over time, which makes the amortized performance penalty for choosing a lower active partition timeout zero.